addon-test-support/@ember/test-helpers/-constants.ts

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
Lets move this into `click.ts` and export from there (still allows us to share, but doesn't require this extra mental step of digging into a separate file)